code/__DEFINES/dcs/signals/signals_mob/signals_mob_main.dm

```diff
@@ -2,3 +2,5 @@
 #define COMSIG_MOB_CLIENT_PRE_LIVING_MOVE "mob_client_pre_living_move"
 	/// Should we stop the current living movement attempt
 	#define COMSIG_MOB_CLIENT_BLOCK_PRE_LIVING_MOVE COMPONENT_MOVABLE_BLOCK_PRE_MOVE
+
+#define COMSIG_MOB_MOVED "mob_moved"
```

Comment:
What's the point of this? We already have `COMSIG_MOVABLE_MOVED` for all the movables (that includes mobs) and otherwise `COMSIG_MOB_CLIENT_MOVED` should be ported for mobs that are controlled by a client (player)